src/table/virtualized-table/hooks/use-data/utils/index.ts

```diff
@@ -15,11 +16,16 @@ const createRow = (
   const pageRowIdx = pageRowStartIdx + cellRowIdx;
   const row: Row = rows[pageRowIdx] ?? { key: `row-${pageRowIdx}` };
 
+  const getBodyCellAlign = (textAlign: 'auto' | Align, cell: EngineAPI.INxCell) =>
```

Comment:
Why is this function created inside the `createRow` function? It looks pure to me?

Reply:
```
const getBodyCellAlign = (textAlign: 'auto' | Align, cell: EngineAPI.INxCell) =>
  textAlign === 'auto' ? autoAlign(cell) : textAlign;

const createRow = (
```

is it okay? or should I move it to another file?

Reply:
It's OK